client/me/purchases/manage-purchase/purchase-meta.jsx

```diff
@@ -0,0 +1,345 @@
+import { getUser } from 'state/users/selectors';
+import paths from '../paths';
+import PaymentLogo from 'components/payment-logo';
+import support from 'lib/url/support';
+import UserItem from 'components/user';
+import {
+	canEditPaymentDetails,
+	isDataLoading,
+	getEditCardDetailsPath,
+	getPurchase,
+	getSelectedSite
+} from '../utils';
+
+class PurchaseMeta extends Component {
+	static propTypes = {
+		hasLoadedSites: React.PropTypes.bool.isRequired,
+		hasLoadedUserPurchasesFromServer: React.PropTypes.bool.isRequired,
+		purchaseId: React.PropTypes.oneOfType( [
+			React.PropTypes.number,
+			React.PropTypes.bool
+		] ).isRequired,
+		selectedPurchase: React.PropTypes.object,
+		selectedSite: React.PropTypes.oneOfType( [
+			React.PropTypes.object,
+			React.PropTypes.bool,
+			React.PropTypes.undefined
+		] )
+	}
+
+	static defaultProps = {
+		hasLoadedSites: false,
+		hasLoadedUserPurchasesFromServer: false,
+		purchaseId: false,
+	}
+
+	renderPrice() {
+		const { translate } = this.props;
+		const purchase = getPurchase( this.props );
+		const { amount, currencyCode, currencySymbol, productSlug } = purchase;
+		const period = productSlug && isMonthly( productSlug ) ? translate( 'month' ) : translate( 'year' );
+
+		if ( isOneTimePurchase( purchase ) ) {
+			return translate( '%(currencySymbol)s%(amount)f %(currencyCode)s {{period}}(one-time){{/period}}', {
+				args: { amount, currencyCode, currencySymbol },
+				components: {
+					period: <span className="manage-purchase__time-period" />
+				}
+			} );
+		}
+
+		if ( isIncludedWithPlan( purchase ) ) {
+			return translate( 'Free with Plan' );
+		}
+
+		return translate( '%(currencySymbol)s%(amount)f %(currencyCode)s {{period}}/ %(period)s{{/period}}', {
+			args: {
+				amount,
+				currencyCode,
+				currencySymbol,
+				period
+			},
+			components: {
+				period: <span className="manage-purchase__time-period" />
+			}
+		} );
+	}
+
+	renderRenewsOrExpiresOnLabel() {
+		const purchase = getPurchase( this.props );
+		const { translate } = this.props;
+
+		if ( isExpiring( purchase ) || creditCardExpiresBeforeSubscription( purchase ) ) {
+			if ( isDomainRegistration( purchase ) ) {
+				return translate( 'Domain expires on' );
+			}
+
+			if ( isSubscription( purchase ) ) {
+				return translate( 'Subscription expires on' );
+			}
+
+			if ( isOneTimePurchase( purchase ) ) {
+				return translate( 'Expires on' );
+			}
+		}
+
+		if ( isExpired( purchase ) ) {
+			if ( isDomainRegistration( purchase ) ) {
+				return translate( 'Domain expired on' );
+			}
+
+			if ( isSubscription( purchase ) ) {
+				return translate( 'Subscription expired on' );
+			}
+
+			if ( isOneTimePurchase( purchase ) ) {
+				return translate( 'Expired on' );
+			}
+		}
+
+		if ( isDomainRegistration( purchase ) ) {
+			return translate( 'Domain renews on' );
+		}
+
+		if ( isSubscription( purchase ) ) {
+			return translate( 'Subscription renews on' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **19** times:
`translate( 'Subscription auto-renews on' )` *ES Score: 10*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).